.claude/skills/changelog-collect/SKILL.md

```
@@ -0,0 +1,328 @@
+```
+
+过滤规则：
+
+- 排除包含 `experimental`、`alpha`、`resource` 的 tag
+- 保留最近的 20 个有效版本 tag
+
+#### 1.2 选择起始版本和目标分支
+
+交互式询问用户：
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)